PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp

```diff
@@ -222,7 +222,7 @@ void VerticalFileSwitcherListView::setItemIconStatus(BufferID bufferID)
 	LVITEM item;
 	item.pszText = fn;
 	item.iSubItem = 0;
-	item.iImage = buf->getUserReadOnly()||buf->getFileReadOnly()?2:(buf->isDirty()?1:0);
+	item.iImage = buf->getUserReadOnly()?3:buf->getFileReadOnly()?2:(buf->isDirty()?1:0);

```

Comment:
`(buf->getUserReadOnly() || buf->getFileReadOnly())` -> Both are read only, so it's the same icon.
You need use another way to detect monitoring status.

Reply:
Do you think I should replace the int number with an enumeration or it isn't useful here?
I will make a method isReadOnly() like in the Notepad_plus.cpp.

Reply:
My bad the method isReadOnly() is already there ^^

Reply:
@ErwanDelhoveUCL

After applying your PR, when I set document to ReadOnly, the icon become monitoring:

![image](https://user-images.githubusercontent.com/90293/97119669-06734580-1712-11eb-9f38-78efbc860e8d.png)